src/kv/kv_types.h

```diff
@@ -31,6 +36,8 @@ namespace kv
   using BatchVector = std::vector<
     std::tuple<kv::Version, std::shared_ptr<std::vector<uint8_t>>, bool>>;
 
+  using SerialisedEntry = std::vector<uint8_t>;
```

Comment:
This was `kv::experimental::SerialisedRep`. I decided `SerialisedEntry` was a clearer name, and it gets promoted to `kv::` as `experimental::` dies.